File: Codes/2798-number-of-employees-who-met-the-target.py
'''
Author: LetMeFly
Date: 2024-04-30 10:37:01
LastEditors: LetMeFly
LastEditTime: 2024-04-30 10:37:03
'''
from typing import List

class Solution:
    def numberOfEmployeesWhoMetTarget(self, hours: List[int], target: int) -> int:
        return sum(t >= target for t in hours)

File: Codes/2798-number-of-employees-who-met-the-target.cpp
/*
 * @Author: LetMeFly
 * @Date: 2024-04-30 10:35:38
 * @LastEditors: LetMeFly
 * @LastEditTime: 2024-04-30 10:35:39
 */
#ifdef _WIN32
#include "_[1,2]toVector.h"
#endif

class Solution {
public:
    int numberOfEmployeesWhoMetTarget(vector<int>& hours, int target) {
        int ans = 0;
        for (int t : hours) {
            ans += t >= target;
        }
        return ans;
    }
};